app/src/main/res/layout/product_list_item.xml

```diff
@@ -39,6 +52,12 @@
                 android:layout_height="wrap_content"
                 tools:text="BBQ Bacon Cheeseburger"
                 tools:textColor="@color/red_cinnabar" />
+            <TextView
```

Comment:
On the image attached, 2nd from the last item in list the description is cut off, also refractor this class to use Constraint layout to avoid nested views. Flat UI is optimal for performance